hazelcast/proxy/ringbuffer.py

```diff
@@ -11,9 +11,30 @@
 
 
 class Ringbuffer(PartitionSpecificProxy):
+    """
+    A Ringbuffer is a data-structure where the content is stored in a ring like structure. A ringbuffer has a capacity
+    so it won't grow beyond that capacity and endanger the stability of the system. If that capacity is exceeded, than
+    the oldest item in the ringbuffer is overwritten. The ringbuffer has 2 always incrementing sequences:
+        #. tail_sequence: this is the side where the youngest item is found. So the tail is the side of the ringbuffer
```

Comment:
(To be consistent, let's use "Ringbuffer" for the rest. (not ring buffer, ringbuffer or RingBuffer).